server/config/deploy.rb

```diff
@@ -0,0 +1,109 @@
+require "mina/rails"
+require "mina/git"
+require "mina/bundler"
+require "mina/multistage"
+
+# require 'mina/rbenv'  # for rbenv support. (https://rbenv.org)
+require "mina/rvm" # for rvm support. (https://rvm.io)
+require "mina/puma"
+require "mina_sidekiq/tasks"
+
+# Basic settings:
+#   domain       - The hostname to SSH to.
+#   deploy_to    - Path to deploy into.
+#   repository   - Git repo to clone from. (needed by mina/git)
+#   branch       - Branch name to deploy. (needed by mina/git)
+
+set :application_name, "ckb-explorer"
+set :domain, "47.97.171.140"
+set :deploy_to, "/home/deploy/ckb-explorer"
+set :repository, "https://github.com/shaojunda/ckb-explorer.git"
+set :branch, "develop"
+set :rails_env, "production"
+set :user, "deploy"
+set :rvm_use_path, "/usr/share/rvm/scripts/rvm"
```

Comment:
Remove this line.

Reply:
why? without this will failed.

Reply:
![image](https://user-images.githubusercontent.com/5165454/56563754-14a54a80-65df-11e9-9a32-1453cf816182.png)

Reply:
Move to staging file, it's not rvm default path, rvm default is `/home/username/.rvm/bin/rvm`, only installed by root will get this path and mina can't find correctly.

Reply:
ok, thank you.